src/App.tsx

```diff
@@ -158,18 +158,10 @@ function AppContent() {
   const { isDesktop, type } = useDeviceInfo();
   const classes = useStyles({ isDesktop });
   const location = useStateLocation();
-  const {
-    organizations,
-    selectedOrganization,
-    reloadData,
-    reloadPreferences,
-    orgPreferences,
-    orgPreferenceForId,
-    userPreferences,
-  } = useOrganization();
+  const { organizations, selectedOrganization, reloadData, orgPreferences, orgPreferenceForId } = useOrganization();
```

Comment:
suggestion while we are cleaning this up, rename `reloadData` to `reloadOrganizations`?